app/components/datafruits-player.js

```diff
@@ -78,7 +78,7 @@ export default Component.extend({
       audioTag.pause();
       this.set('playButtonPressed', false);
       this.set('playerState', 'paused');
-      if(this.playingPodcast === false){
+      if(this.playingPodcast === false || this.isLive){
```

Comment:
I'm not sure this is needed here. If we're playing *not* playing a podcast, it should always be a live stream that's playing instead.

Reply:
Addressed in 
7839b97